vendor/magento/zendframework1/library/Zend/Locale/Data/nl_SR.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
We should not push changes on the vendor folder, that is handled by composer.

Reply:
Then how can I update the changes related to the vendor?

Reply:
@aasim110: the zf1 repo is located here: https://github.com/magento/zf1/
You can try to send in a PR over there.
But the response times over there are much worse than they are on the magento2 repo (@gabrieldagama: hint hint, maybe somebody could have a look at the open PR's there)